scripts/utils/index.ts

```diff
@@ -146,7 +156,9 @@ export const deployContract = async (
   deployerInstance: Deployer
 ): Promise<string> => {
   //const { hash, wait } = await deployerInstance.deploy(salt, contractByteCode, {maxFeePerGas: 200e9, maxPriorityFeePerGas: 75e9});
-  const { hash, wait } = await deployerInstance.deploy(salt, contractByteCode, {gasPrice: 40e9});
+  const { hash, wait } = await deployerInstance.deploy(salt, contractByteCode, {
+    gasPrice: 40e9,
```

Comment:
this prices also we need to make it network wise.

Reply:
added